src/services/refactors/extractMethod.ts

```diff
@@ -909,7 +909,7 @@ namespace ts.refactor.extractMethod {
         }
     }
 
-    function getStatementsOrClassElements(scope: Scope): ReadonlyArray<Statement> | ReadonlyArray<ClassElement> {
+    function getStatementsOrClassElements(scope: Scope): ReadonlyArray<Statement | ClassElement> {

```

Comment:
I don't believe these types are equivalent.  Didn't the old version meant that the list was either all statements or all class elements?

Reply:
Yes, but we don't need that information and it breaks type information on `find`.

Reply:
Personally, I would leave a comment explaining why we're not using the more accurate type.